week06/6.1.find_special_state.py

```diff
@@ -0,0 +1,36 @@
+#=======================
+# שטוחלנדיה - 6.1
+#=======================
+
+
+# set of the letters in the 1st row of the keyboard
+topRow = set("qwertyuiop")
+
+# set of the letters in the 2nd row of the keyboard
+midRow = set("asdfghjkl")
+
+# set of the letters in the 3rd row of the keyboard
+botRow = set("zxcvbnm")
```

Comment:
Python is snake_case https://www.geeksforgeeks.org/python-program-to-convert-camel-case-string-to-snake-case/

Reply:
Fixed now